ddtrace/_trace/processor/__init__.py

```diff
@@ -250,6 +252,8 @@ def process_trace(self, trace: List[Span]) -> Optional[List[Span]]:
             span._update_tags_from_context()
             self._set_git_metadata(span)
             span._set_tag_str("language", "python")
+            if serialized_process_tags := process_tags.get_serialized_process_tags():
```

Comment:
this should basically be static, no? any way we can just avoid the function call here entirely?

Reply:
addressed in [dd58490](https://github.com/DataDog/dd-trace-py/pull/15146/commits/dd584901b5ca43f568cd079052a90e35011b579c)